org.freedesktop.Platform.GL.nvidia.json.in

```diff
@@ -22,6 +23,25 @@
         "--metadata=Extra Data=NoRuntime"
     ],
     "modules": [
+        {
+            "name": "libzstd",
```

Comment:
This must be a static library like the others are.

Reply:
It appears you aren't using the static one though as you are using `append-ld-library-path`?

Reply:
`append-ld-library-path` was added so `libarchive` could find the `zstd` libraries we built.

If we don't strip the final binary (`apply_extra`), we can verify that it contains all symbols from `zstd`:

```
$ nm -an apply_extra | c++filt | grep ZSTD
00000000004004a4 t ZSTD_decompressLegacyStream
0000000000425720 T ZSTD_versionNumber
0000000000425730 T ZSTD_versionString
0000000000425740 T ZSTD_isError
0000000000425750 T ZSTD_getErrorName
0000000000425770 T ZSTD_getErrorCode
...
```

On my machine I was able to build, install and extract the 530.30.02 drivers normally (although I haven't particularly tested if they work, because my old NVIDIA GPU doesn't support these new drivers).

Reply:
Alright, thanks for clarifying